libopae/plugins/xfpga/sysobject.c

```diff
@@ -47,6 +47,13 @@
 		}                                                              \
 	} while (false);
 
+#define VALIDATE_NAME(_N)                                                      \
+	do {                                                                   \
+		if (_N[0] == '.' || _N[0] == '/') {                            \
```

Comment:
What if .. appears, but not as the first character, eg "errors/../../../../../../../../../../../"?